README.md

```diff
@@ -353,21 +353,21 @@ A node attribute MUST be one of these:
     <th>Direction</th>
     <th>Description</th>
     <th>Retained</th>
-    <th>Required</th>
+    <th>Required (default)</th>
   </tr>
   <tr>
     <td>$name</td>
     <td>Device → Controller</td>
     <td>Friendly name of the Node</td>
     <td>Yes</td>
-    <td>Yes</td>
+    <td>No (the node ID is used)</td>
```

Comment:
Sounds like the view by someone who implements the controller side of the convention 😃 
From the convention side of things: Do we want to give such promises or do we want to leave this decision to the controller?

Reply:
Yes I'm not sure about this one. It potentially saves space on the device if the node is called "Temperature" and the name would be the same.